scripts/dts/extract/globals.py

```diff
@@ -331,6 +331,56 @@ def build_cell_array(prop_array):
 
     return ret_array
 
+def child_to_parent_unmap(cell_parent, gpio_index):
+    # This function returns a (gpio-controller, pin number) tuple from
+    # cell_parent (identified as a 'nexus node', ie: has a 'gpio-map'
+    # property) and gpio_index.
+    # Note: Nexus nodes and gpio-map property are described in the
+    # upcoming (presumably v0.3) Device Tree specification, chapter
+    # 'Nexus nodes and Specifier Mapping'.
+
+    # First, retrieve gpio-map as a list
+    raw_map = reduced[cell_parent]['props']['gpio-map']
```

Comment:
Do you think this could be called `gpio_map`?

Reply:
It could